development-docs/systemtests/io.strimzi.systemtest.operators.user.UserST.md

```diff
@@ -0,0 +1,136 @@
+# UserST
+
+**Description:** Test suite for various Kafka User operations and configurations.
+
+**Before tests execution steps:**
+
+| Step | Action | Result |
+| - | - | - |
+| 1. | Initialize shared Test Storage and deploy Kafka cluster with necessary configuration. | Kafka cluster and scraper pod are deployed and ready for testing. |
+
+**Labels:**
+
+* `user-operator` (description file doesn't exist)
+
+<hr style="border:1px solid">
+
+## testCreatingUsersWithSecretPrefix
+
+**Description:** Verifies creating Kafka users with custom secret prefixes for organizing user secrets.
+
+**Steps:**
+
+| Step | Action | Result |
+| - | - | - |
+| 1. | Configure cluster operator with custom secret prefix. | Cluster operator is reconfigured with the specified secret prefix. |
+| 2. | Create TLS and SCRAM-SHA users. | Users are created successfully with TLS and SCRAM-SHA authentication. |
+| 3. | Verify user secrets are created with proper prefix. | User secrets are created with the configured prefix in their names. |
+| 4. | Test message sending and receiving. | Messages are successfully sent and received using both authentication methods. |
+| 5. | Update users and verify secret updates. | User updates are reflected in the prefixed secrets. |
+| 6. | Delete users and verify cleanup. | User deletion removes the prefixed secrets properly. |
+
+**Labels:**
+
+* `user-operator` (description file doesn't exist)
+
+
+## testScramUserWithQuotas
+
+**Description:** Verifies that SCRAM-SHA authenticated Kafka users can be configured with quotas.
```

Comment:
Shall we say SCRAM-SHA-512 (replacing SCRAM-SHA and SCRAM only) to be consistent throughout?

Reply:
Sure, I can do it.